src/app/Http/Controllers/UserController.php

```diff
@@ -2,13 +2,20 @@
 
 namespace App\Http\Controllers;
 
+use Illuminate\Http\RedirectResponse;
 use Illuminate\View\View;
 
 class UserController extends Controller
 {
-    public function profile(): View
+    public function show(): View
     {
-        return view('user.profile');
+        return view('user/profile');
+    }
+
+    public function logout(): RedirectResponse
+    {
+        auth()->logout();
+        return redirect('/');
```

Comment:
Same goes here, if  you have a declared route name for '/' you should use that instead